tests/Aspire.Hosting.Containers.Tests/Aspire.Hosting.Containers.Tests.csproj

```
@@ -11,6 +11,10 @@
     <ProjectReference Include="..\Aspire.Hosting.Tests\Aspire.Hosting.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)src\Aspire.Hosting.Testing\ResourceLoggerForwarderService.cs" Link="Utils\ResourceLoggerForwarderService.cs" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Polly.Core" />
   </ItemGroup>
```

```suggestion
  <ItemGroup>
    <PackageReference Include="Polly.Core" />
  </ItemGroup>

    <ItemGroup>
    <Compile Include="$(RepoRoot)src\Aspire.Hosting.Testing\ResourceLoggerForwarderService.cs" Link="Utils\ResourceLoggerForwarderService.cs" />
  </ItemGroup>
```

(nit) I would keep the Package and Project references together, and not split them apart with a `Compile` section.